contrib/buildahimage/upstream/Containerfile

```
@@ -17,7 +17,10 @@ ENV GOPATH=/root/buildah
 # to the container.
 # Finally remove the buildah directory and a few other packages
 # that are needed for building but not running Buildah
-RUN useradd build; yum -y update;  rpm --restore shadow-utils 2>/dev/null; yum -y install --enablerepo=updates-testing \
+RUN useradd build; \
+    dnf -y update;  && \
```

```suggestion
    dnf -y update && \
```

This is causing a build-failure.